docs/pages/pmd/languages/java_metrics_index.md

```diff
@@ -62,6 +62,76 @@ public class Foo { // total 8
 
 * Option `includeJavaLang`: Also include classes from the package `java.lang`
 
+## Cognitive Complexity (COGNITIVE_COMPLEXITY)
+
+*Operation metric.* Can be calculated on any non-abstract operation.
+
+### Description
+
+Cognitive complexity is a measure of how difficult it is for humans to read and understand a method. Code that contains
+a break in the control flow is more complex, whereas the use of language shorthands doesn't increase the level of
+complexity. Nested control flows can make a method more difficult to understand, with each additional nesting of the
+control flow leading to an increase in cognitive complexity.
+
+Information about Cognitive complexity can be found in the original paper here:
+[CognitiveComplexity](https://www.sonarsource.com/docs/CognitiveComplexity.pdf)
+
+By default, this rule reports methods with a complexity of 15 or more. Reported methods should be broken down into less
```

Comment:
Strictly speaking, this sentence belongs to the rule and not to the metric. We could refer to the rule here, e.g.:

```suggestion
The rule {% rule java/design/CyclomaticComplexity %} by default reports methods with a complexity of 15 or more. 
These reported methods should be broken down into less
```